python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py

```diff
@@ -157,6 +136,37 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
     )
 
     assert result.success
+    return result
+
+
+def test_materialize_no_selection(dbt_commands):
+    result = _materialize_dbt_assets(dbt_commands, None)
```

Comment:
types + explicit kwargs for reading purposes
```suggestion
def test_materialize_no_selection(dbt_commands: List[List[str]]) -> None:
    result = _materialize_dbt_assets(dbt_commands, selection=None)
```